ivre/utils.py

```diff
@@ -2112,7 +2119,11 @@ def get_cert_info(cert: bytes) -> Dict[str, Any]:
             hashtype: hashlib.new(hashtype, cert).hexdigest()
             for hashtype in ["md5", "sha1", "sha256"]
         }
-        data = osslc.load_certificate(osslc.FILETYPE_ASN1, cert)
+        try:
+            data = osslc.load_certificate(osslc.FILETYPE_ASN1, cert)
+        except OpenSSL.crypto.Error:
```

Comment:
Already imported as:
```suggestion
        except osslc.Error:
```